nettrine/app/Presenters/DoctrineExtensionsPresenter.php

```diff
@@ -0,0 +1,102 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App\Model\Database\Basic\Entity\Book;
+use App\Model\Database\Basic\Entity\Category;
+use App\Model\Database\EntityManagerDecorator;
+use App\Model\Database\Translatable\Entity\Article;
+use Gedmo\Loggable\Entity\LogEntry;
+use Nette\Application\UI\Form;
+use Nette\Application\UI\Presenter;
+
+class DoctrineExtensionsPresenter extends Presenter
+{
+
+	/** @var EntityManagerDecorator @inject */
+	public $em;
+
+	public function renderDefault(): void
+	{
+		$articleRepository = $this->em->getArticleRepository();
+		$articlesEN = [];
+
+		foreach ($articleRepository->findAll() as $article) {
```

Comment:
Co vlastne dela toto? Vsiml jsem si, ze tady je en_GB a cs_CZ. Proc je to tu dvakrat?

Reply:
![obrazek](https://user-images.githubusercontent.com/6066243/89068879-65732980-d372-11ea-8703-571d18a85067.png)

Reply:
protože dvakrát vypisuju, viz náhled